mypy/test/testcheck.py

```diff
@@ -85,6 +85,9 @@
     'check-newsemanal.test',
 ]
 
+if (3, 8) < sys.version_info:
```

Comment:
I would flip this.
```suggestion
if  sys.version_info >= (3, 8):
```
Also please add explanatory comment.